pytest.ini

```diff
@@ -0,0 +1,10 @@
+[pytest]
+filterwarnings =
+    ignore:Call to deprecated create function Descriptor
```

Comment:
I can see these are coming from the google api, i wonder if we can exclude that from the check rather than adding these here

Reply:
It's a temporary problem arising from protobuf not having a Python 3.9 wheel yet. Removing pytest.ini: anyone testing frontstage can put these lines into their setup.cfg if they wish, but it's only a minor annoyance and will likely be fixed soon.